sigopt/lib.py

```diff
@@ -56,3 +56,6 @@ def find(lis, predicate):
   Finds the first element in lis satisfying predicate, or else None
   """
   return next((item for item in lis if predicate(item)), None)
+
+def api_url_to_app_url(api_url):
+  return ':'.join(api_url.replace('//api.', '//app.').split(':')[:2])
```

Comment:
This may or may not work, depending on the url structure - we have things like onprem that might have totally different URLs for API and APP, it doesn't have to follow the paradigm we happen to use.

Instead, the same way we define the `api_url` through things like environment variables (`SIGOPT_API_URL`) or explicit calls (`set_api_url`), we should do something similar for `app_url` without tying it directly to the API url structure

Reply:
Yes, it may not work. I think it a quick fix and works by now. We may need a new `SIGOPT_APP_URL`.

Reply:
Yeah, espec since this is a public-facing repo, and the fix would likely only apply to dev / special environments, we don't want to merge in any "may not work" changes. But `SIGOPT_APP_URL` would be great and robust, so if you wanna work on that instead (can use the other `SIGOPT_API_*` variables as inspiration), that would be great

Reply:
For local development environments, `SIGOPT_APP_URL` should be sufficient for this purpose.

For users in the on-prem setting, it would probably be more appropriate to redirect to the web using the api url (ex. `https://api.sigopt.ninja/app/... => 301 https://app.sigopt.ninja`) but the appropriate nginx routes have not been implemented.

The redirect feature that supports on-prem is not likely to be prioritized, so it is probably a good decision to continue with `SIGOPT_APP_URL`.